.github/utils/get_dependency_releases.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+import requests
+from tabulate import tabulate
+
+if len(sys.argv) < 2:
+    print(f'Usage: python {sys.argv[0]} <list of packages>')
+    sys.exit(1)
+
+packages = sys.argv[1:]
+session = requests.Session()
+
+releases = []
+for package in packages:
+    pypi = session.get(f'https://pypi.org/pypi/{package}/json').json()
+    latest_version = pypi['info']['version']
```

Comment:
If this is not a 200, we could get a mystifying KeyError here. I don't think we need to handle it super well, since it's a utility, but just a heads up.

Reply:
Good call. I will have it raise before we dig into the dict

Reply:
Fixed